pkg/ddevapp/wordpress.go

```diff
@@ -182,37 +217,68 @@ func WriteWordpressConfig(wordpressConfig *WordpressConfig, filePath string) err
 	if err != nil {
 		return err
 	}
-	err = tmpl.Execute(file, wordpressConfig)
+	defer util.CheckClose(file)
+
+	if err = tmpl.Execute(file, wordpressConfig); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// writeWordpressDdevSettingsFile unconditionally creates the file that contains ddev-specific settings.
+func writeWordpressDdevSettingsFile(config *WordpressConfig, filePath string) error {
+	tmpl, err := template.New("wordpressConfig").Funcs(sprig.TxtFuncMap()).Parse(wordpressDdevSettingsTemplate)
+	if err != nil {
+		return err
+	}
+
+	// Ensure target directory exists and is writable
+	dir := filepath.Dir(filePath)
+	if err = os.Chmod(dir, 0755); os.IsNotExist(err) {
+		if err = os.MkdirAll(dir, 0755); err != nil {
+			return err
+		}
+	} else if err != nil {
+		return err
+	}
+
+	file, err := os.Create(filePath)
 	if err != nil {
 		return err
 	}
-	util.CheckClose(file)
+	defer util.CheckClose(file)
+
+	if err = tmpl.Execute(file, config); err != nil {
+		return err
+	}
+
 	return nil
 }
 
 // setWordpressSiteSettingsPaths sets the expected settings files paths for
 // a wordpress site.
 func setWordpressSiteSettingsPaths(app *DdevApp) {
+	config := NewWordpressConfig(app)
+
 	settingsFileBasePath := filepath.Join(app.AppRoot, app.Docroot)
-	var settingsFilePath, localSettingsFilePath string
-	settingsFilePath = filepath.Join(settingsFileBasePath, "wp-config.php")
-	localSettingsFilePath = filepath.Join(settingsFileBasePath, "wp-config-local.php")
-	app.SiteSettingsPath = settingsFilePath
-	app.SiteLocalSettingsPath = localSettingsFilePath
+	app.SiteSettingsPath = filepath.Join(settingsFileBasePath, config.SiteSettings)
+	app.SiteLocalSettingsPath = filepath.Join(settingsFileBasePath, config.SiteSettingsLocal)
 }
 
 // isWordpressApp returns true if the app of of type wordpress
 func isWordpressApp(app *DdevApp) bool {
-	if _, err := os.Stat(filepath.Join(app.AppRoot, app.Docroot, "wp-login.php")); err == nil {
-		return true
+	_, err := getRelativeAbsPath(app)
+	if err != nil {
+		return false
 	}
-	return false
+
+	return true
 }
 
 // wordpressPostImportDBAction just emits a warning about updating URLs as is
```

Comment:
Let's just remove this since it does nothing. There's no need to keep wordpressPostImportDBAction

Reply:
Sure, removed.